spec/features/thredded/user_creates_messageboard_group_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+feature 'Creating a messageboard group' do
+  scenario 'admin creates a unique messageboard group' do
+    user = an_admin
+    user.log_in
+
+    group = set_up_a_messageboard_group
+    group.visit_new_messageboard_group_form
+    group.submit_form
+
+    expect(group).to be_created
+  end
+
+  scenario 'admin creates non unique messageboard group' do
+    user = an_admin
+    user.log_in
+
+    group = set_up_a_messageboard_group
+    group.create_messageboard_group
+    group.visit_new_messageboard_group_form
+    group.submit_form
+
+    expect(group).to be_duplicate
+  end
```

Comment:
No, the test is looking for an error message that it didn't get created.

I'd say you never want a dupe, that's what the PR is preventing